packages/assets/assets/spinner-error.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
No need to have this icon. Ziad will have another design at which the user will open the actions menu and click re-send.